src/components/App/index.js

```diff
@@ -1,3 +1,13 @@
-import App from './App';
+import React from 'react';
+
+import Item from '../Item/Item';
+
+function App() {
+  return (
+    <section>
```

Comment:
I think you merged with master incorrectly. I think we had routes already configured in App.
Good place to put this Item component is Cart component.